ui/src/stores/indexedDB.ts

```
@@ -0,0 +1,128 @@
+};
+
+export type DebugRunState = {
+  jobs: JobState[];
+};
+
+export type PreferencesState = {
+  theme: string;
+};
+
+export type AppState = {
+  [GENERAL_KEY]: GeneralState;
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)